src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IBinaryOperatorExpression.cs

```diff
@@ -414,6 +414,32 @@ void M(int a, int b, int c, int d, int e, int f, int g, int h, int i, int j, int
             VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestBinaryOperators_UnsignedRightShift()
```

Comment:
Should we also have IOp tests with `>>>=`, or will that be a later PR?

Reply:
> Should we also have IOp tests with ```>>>=```, or will that be a later PR?

Yes we should have those. I will add them in a later PR.